docs/DeveloperGuide.md

```diff
@@ -564,6 +564,47 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: Delete a tag**
```

Comment:
```suggestion
#### Use case: Delete a tag
```
I think we should change to using headers instead of bolding as it can be hard to separate the use cases?